### PR TITLE
[pxTextBox] Improve multibyte character handling

### PR DESCRIPTION
**Pull request by ashwinikalokhe**

### CODE REVIEW

File: examples/pxScene2d/src/pxTextBox.cpp [hunk 1]
```diff
@@ -388,25 +388,14 @@ void pxTextBox::measureTextWithWrapOrNewLine(const char *text, float sx, float s
     int i = 0;
     int lasti = 0;
     int numbytes = 1;
-    char* tempChar = NULL;
+    //char* tempChar = NULL;
```
Comment (mfiess) [inline examples/pxScene2d/src/pxTextBox.cpp]: Please remove this code instead of commenting it out (since it is no longer needed).

Comment (ashwinikalokhe) [reply examples/pxScene2d/src/pxTextBox.cpp]: Thanks Mike. I have cleaned this code.

File: examples/pxScene2d/src/pxTextBox.cpp [hunk 2]
```diff
@@ -522,13 +511,13 @@ void pxTextBox::measureTextWithWrapOrNewLine(const char *text, float sx, float s
...
-              accString.append(tempChar);
+              accString.append(tempChar.c_str());
             }
             
           }
 
-          delete [] tempChar;
-          tempChar = NULL;
+          /*delete [] tempChar;
+          tempChar = NULL;*/
```
[the remark this answers is not in the document]
Comment (ashwinikalokhe) [reply examples/pxScene2d/src/pxTextBox.cpp]: Done

File: examples/pxScene2d/src/pxTextBox.cpp [hunk 3]
```diff
@@ -557,11 +546,11 @@ void pxTextBox::measureTextWithWrapOrNewLine(const char *text, float sx, float s
         }
       }
     }//WHILE
-    if (tempChar != NULL)
+    /*if (tempChar != NULL)
     {
       delete [] tempChar;
       tempChar = NULL;
-    }
+    }*/
```
[the remark this answers is not in the document]
Comment (ashwinikalokhe) [reply examples/pxScene2d/src/pxTextBox.cpp]: Done

**Pull request merged**